CMSgov/design-system
# [WNMGDS-827] remove drawer heading focus

## Summary

Remove `tabindex` from non-interactive elements in Drawer. Original ticket specified the heading element, but I also saw this attribute was added to the body `<div>` of the Drawer (and did nothing for focus anyway).

## How to test

1. Pull this branch and build Storybook locally
2. View http://localhost:6006/?path=/story/components-drawer--drawer-toggle-with-drawer and check other Drawer stories to see focus no longer applied to text elements

## Checklist




### If this is a change to code:
